### ISSUE TITLE
Grave didn't generate on Quark basalt (unable to reproduce)

### ISSUE THREAD

**Issue by Daidez** [M4thG33k/TombManyGraves#34]
I died in the nether while standing on top of Quark basalt, my grave did not generate, and I lost all of my items. I looked around for my grave, but I couldn't find it. I was playing on a server when all of this happened.

I tried to reproduce the issue in singleplayer creative mode, and it generated the grave properly this time.
(Server-related issue...?)

**Comment by M4thG33k** [comment]
Without being able to reproduce it, it'll be hard to track down the issue.

A server op may be able to restore your inventory from a backup file.

**Comment by Daidez** [comment]
It might be server-related, (not sure why it would be) because it worked fine in SP.

I'll ask the owner about restoring my inventory.

Thanks for looking at my issue.

**Comment by M4thG33k** [comment]
If you can get the server log for that time from an admin, that might help me out.

**Comment by Daidez** [comment]
I'll see what I can do.

**Comment by scederqv** [comment]
Hi, server admin here. The only logs I could find of the incident is:

[23:46:03] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save world
[23:46:03] [Server thread/DEBUG] [FluxNetworks/]: All 1 Networks were saved successfully
[23:46:07] [Server thread/INFO] [TombManyGraves/]: [TombManyGraves]: Daidez died in dimension -1 at (x,y,z) = (17,36,44). Their grave may be near! 
[23:46:09] [Server thread/INFO] [journeymap/]: Daidez joining dimension 0 [23:46:09] [Server thread/INFO] [mcjty.lib.varia.Logging/]: TileEntity is not a CommandHandler! 
[23:46:14] [Server thread/DEBUG] [FML/]: The world 6bc22b8d (world) may have leaked: seen 4535 times.

Afraid I cannot confirm anything by that really.

**Comment by M4thG33k** [comment]
Yeah, that tells me that the grave should have formed. Was a Death List created when you respawned @Daidez?

**Comment by M4thG33k** [comment]
Weird. But I can't really do anything without being able to replicate the issue. :-/  The grave is supposed to default to spilling everything onto the ground if it can't form properly (and will say with a chat message "Could not find suitable grave location"), so this is definitely weird. If anyone can replicate this, please let me know.
